apps/server/src/modules/h5p-editor/contentStorage/contentStorage.spec.ts

```diff
@@ -204,10 +308,7 @@ describe('ContentStorage', () => {
 			it('should add a file to content', async () => {
 				const { stream, contentId } = setup();
 				const filename = 'testFiletoAdd123.json';
-				await service.addFile(contentId, filename, stream);
-				await delay(0);
-				const fileExists = fs.existsSync(path.join(dir, contentId.toString(), filename));
-				expect(fileExists).toEqual(true);
+				expect(await service.addFile(contentId, filename, stream)).not.toBeInstanceOf(Error);
```

Comment:
I think you should test here that `storageClient.create` is called.

I does not make sense to test if addFile returns an error. It won't ever _return_ an error. If an error occurs inside it, it will _throw_ the rejection reason. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#exceptions